package.json

```diff
@@ -18,9 +18,11 @@
     "release": "npm version ${SEMANTIC:-patch} -m \"Release %s\" && npm run build && git push && git push --tags && npm publish",
     "test": "npm-run-all build mocha",
     "mocha": "NODE_ENV=test mocha --compilers js:babel-register --reporter spec test/",
-    "build": "babel src/ -d lib/",
+    "build": "./node_modules/babel-cli/bin/babel.js src/ -d lib/",
```

Comment:
This _should_ not be necessary can you check what is in `node_modules/.bin/*`? npm adds that to the `$PATH` so that you can just use the executable's name.